test/integration/monolithic/serverspec/monolithic_config_spec.rb

```diff
@@ -22,6 +22,17 @@
         end
     end
 
+    it 'has websockify service script configured' do
+        expect(file('/etc/init.d/websockify')).to contain("WEBSOCKIFY_PORT=41337")
+        expect(file('/etc/init.d/websockify')).to contain("CERT_FILE=/etc/pki/abiquo/monolithic.abiquo.com.crt")
+        expect(file('/etc/init.d/websockify')).to contain("KEY_FILE=/etc/pki/abiquo/monolithic.abiquo.com.key")
+    end
+
+    it 'has novnc_tokens cron task configured' do
+        expect(file('/etc/corn.d/novnc_tokens')).to_not be_executable
```

Comment:
If we are testing this, we should have a `file` resource in the install_server recipe that enforces this file mode. Otherwise we are testing that are not enforced by the cookbook and could eventually fail.